MyModTest/BaseTest.cs

```diff
@@ -4914,7 +4914,7 @@ protected GameController LoadGame(string name, bool addDataPath = false, bool ad
                 {
                     var dllpath = System.Reflection.Assembly.GetExecutingAssembly().CodeBase;
                     path = Path.GetDirectoryName(dllpath.Replace("file://", "")).Replace("\\D:", "D:").Replace("\\C:", "C:");
-                    path = Path.Combine(path, "..", "..", "DataFiles", name);
+                    path = new string[] { path, "..", "..", "DataFiles", name }.Aggregate((x, y) => Path.Combine(x, y));
```

Comment:
This was the only breaking change between 3.5 and 4.x. `Path.Combine` only supported multiple params in 4.0+